app/models/micropost.rb

```diff
@@ -0,0 +1,19 @@
+class Micropost < ApplicationRecord
+  belongs_to :user
+
+  mount_uploader :picture, PictureUploader
+  scope :newest, -> {order created_at: :asc}
+
+  validates :user_id, presence: true
```

Comment:
because the effect of line number 2, this line doesn't need. You can remove it

Reply:
yes